apps/portal/src/content/docs/components/toggle-group.mdx

```diff
@@ -0,0 +1,215 @@
+
+return (
+  <ToggleGroup.Root onValueChange={updateControlledInputValue} value={controlledInputValue} type="single">
+    <ToggleGroup.Item value="1">Option 1</ToggleGroup.Item>
+    <ToggleGroup.Item value="2">Option 2</ToggleGroup.Item>
+    <ToggleGroup.Item value="3">Option 3</ToggleGroup.Item>
+  </ToggleGroup.Root>
+)
+```
+
+## Anatomy
+
+All parts of the Toggle Group component can be imported and composed as required.
+
+```typescript jsx
+<ToggleGroup.Root variant="compact">
+  <ToggleGroup.Item value="1">Option 1</ToggleGroup.Item>
+  <ToggleGroup.Item value="2">Option 2</ToggleGroup.Item>
+  <ToggleGroup.Item value="3">Option 3</ToggleGroup.Item>
+</ToggleGroup.Root>
+```
+
+## API Reference
+
+### Root
+
+The `Root` component serves as the main container for all toggle group elements.
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "children",
+      description: "You can pass in your `ToggleGroup.Item` as children",
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description: "Additional class names to apply to the root element",
+      required: false,
+      value: "string",
+    },
+    {
+      name: "variant",
+      description: "Variant of the toggle group",
+      required: false,
+      value: "'default' | 'outline' | 'compact'",
+    },
+    {
+      name: "type",
+      description: "Type of the toggle group",
+      required: true,
+      value: "'single' | 'multiple'",
+    },
+    {
+      name: "value",
+      description:
+        "Controlled stateful value of the item(s) that is/are pressed",
+      required: false,
+      value: "string | string[]",
+    },
+    {
+      name: "defaultValue",
+      description:
+        "Value of the item(s) that is/are pressed when initially rendered",
+      required: false,
+      value: "string | string[]",
+    },
+    {
+      name: "onValueChange",
+      description:
+        "Callback that fires when the value of the toggle group changes",
+      required: false,
+      value: "(value: string | string[]) => void",
+    },
+    {
+      name: "disabled",
+      description: "Whether the group is disabled from user interaction",
+      required: false,
+      value: "boolean",
+      defaultValue: "false",
+    },
+    {
+      name: "rovingFocus",
+      description:
+        "Whether the group should maintain roving focus of its buttons",
+      required: false,
+      value: "boolean",
+      defaultValue: "true",
+    },
+    {
+      name: "loop",
+      description: "Whether the focus should loop around when reaching the end",
+      required: false,
+      value: "boolean",
+    },
+    {
+      name: "orientation",
+      description: "Orientation of the toggle group",
+      required: false,
+      value: "'horizontal' | 'vertical'",
+    },
+    {
+      name: "dir",
+      description: "Text direction of the toggle group",
+      required: false,
+      value: "'ltr' | 'rtl'",
```

Comment:
Safe to assume the default is `ltr`? We probably shouldn't expose this prop since we don't support `rtl` languages at the moment.